tests/modules/programs/mpv/mpv-stubs.nix

```diff
@@ -1,26 +1,27 @@
-{ pkgs, ... }:
```

Comment:
The overlay was referencing `pkgs`, which lead to an infinite recursion between `pkgs` and the overlay.